styles/style.css

```diff
@@ -91,12 +91,28 @@ p {
   border: 6px solid var(--white);
   border-left-color: var(--orange-dark);
 }
+.white-text {
+  color: wheat;
```

Comment:
I think you mean "white" here and elsewhere? You could even use the `var{--white)` variable.